domains/authorization/Authorization.API/API/Authorization/[Features]/DeleteConsentCommand.cs

```diff
@@ -0,0 +1,51 @@
+using System;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using API.Data;
+using API.Repository;
+using API.ValueObjects;
+using MediatR;
+using Microsoft.EntityFrameworkCore;
+
+namespace API.Authorization._Features_;
+
+public class DeleteConsentCommandHandler(
+    IConsentRepository consentRepository,
+    IUserRepository userRepository,
+    IUnitOfWork unitOfWork)
+    : IRequestHandler<DeleteConsentCommand, bool>
+{
+    public async Task<bool> Handle(DeleteConsentCommand request, CancellationToken cancellationToken)
```

Comment:
No need to return a boolean

Reply:
Fixed: https://github.com/Energinet-DataHub/energy-origin/pull/1236/commits/94b1f55c3688b382e42989a6820e5bde220f891d